webknossos-datastore/app/com/scalableminds/webknossos/datastore/datareaders/zarr/NgffMetadata.scala

```diff
@@ -107,3 +107,13 @@ object NgffLabelsGroup {
   implicit val jsonFormat: OFormat[NgffLabelsGroup] = Json.format[NgffLabelsGroup]
   val LABEL_PATH = "labels/.zattrs"
 }
+
+case class NgffOmeroObject(channels: List[NgffChannelAttributes])
```

Comment:
```suggestion
case class NgffOmeroMetadata(channels: List[NgffChannelAttributes])
```

I’d go for this name over Object if that’s alright with you :)